Update docker (#64)
opendihu/opendihu: doc/sphinx/user/installation.rst

Using docker
----------------
With the docker image, you can use the framework directly without having to build and install any dependencies. Parallel execution with MPI is possible.

> The disadvantage is that you only have a shell and can't plot anything. However, you may copy the files from the docker container to your machine and do the plotting there. 

Reviewer: ```suggestion
The disadvantage is that you only have a shell and can't plot anything. However, you may copy the files from the docker container to your machine (or directly mount the respective directory) and do the plotting there. 
```

> You can either download a pre-built image or building the container from source using one of the available `Dockerfile`s. 

Reviewer: ```suggestion
You can either download a pre-built image or build an image from source using one of the available `Dockerfile`. 
```

In either case, you will have to install Docker first. For that we refer you to the official `docker installation instructions <https://docs.docker.com/engine/install/>`_ .

* Using the pre-built docker image
  * Pull the image from `DockerHub <https://hub.docker.com/r/carmehp/opendihu/tags>`_ 
    .. code-block:: bash

    docker image pull carmehp/opendihu
  * Run the image
    .. code-block:: bash

>     docker run -i -t carmehp/opendihu

Reviewer: It is common to combine these two rather boilerplate options:

```suggestion
    docker run -it carmehp/opendihu
```

  By default, carmehp/opendihu is equivalent to ``carmehp/opendihu:latest``. If you want to use a different image, then you have to specify its tag: `carmehp/opendihu:<tag_name>`.

* Building the docker image from source
  * Select the image you want to build. All the available options are found in ``tools/docker/``. For example if you want to use the docker image based on Ubuntu 22.04, change to the corresponding directory where the Dockerfile is located.
    .. code-block:: bash

    cd tools/docker/ubuntu22
  * Build the image. The ``<image_name>`` can be choosen freely. This step will take some time, e.g., 30 minutes - 1 hour.
    .. code-block:: bash

    docker build -t <image_name> .
  * Run a container of the image. The ``<image_name>`` can be choosen freely.
    .. code-block:: bash

>     docker run -i -t <image_name>  

Reviewer: Same here:

```suggestion
    docker run -it <image_name>  
```
